packages/Babylonian-UI.package/BPMethodExampleMorph.class/instance/addArgumentScriptMorphs.st

```diff
@@ -6,7 +6,7 @@ addArgumentScriptMorphs
 		container := self newContainerMorph.
 		container cellPositioning: #topLeft.
 		labelMorph := ((argumentName , ': ') asMorph 
-			color: Color lightGray; 
+			color: self class textColor; 
```

Comment:
This is in this method and #addReceiverConstructorMorph; maybe extract into its own method